QueryEngine/NativeCodegen.cpp

```diff
@@ -100,23 +101,34 @@ void optimize_ir(llvm::Function* query_func,
                  llvm::Module* module,
                  const std::unordered_set<llvm::Function*>& live_funcs,
                  const CompilationOptions& co) {
-  llvm::legacy::PassManager pass_manager;
+  if (co.opt_level_ >= ExecutorOptLevel::O1 && co.opt_level_ <= ExecutorOptLevel::O3) {
```

Comment:
Could we pick a consistent naming theme in this function? mpm, pass_manager..

Reply:
Done